plugins/libplugin-pay.h

```diff
@@ -164,7 +209,25 @@ struct payment {
 	void **modifier_data;
 	int current_modifier;
 
+	struct bolt11 *invoice;
+
+	/* tal_arr of channel_hints we incrementally learn while performing
+	 * payment attempts. */
+	struct channel_hint *channel_hints;
+	struct node_id *excluded_nodes;
+
 	struct payment_result *result;
+
+	/* Did something happen that will cause all future attempts to fail?
+	 * This usually means that the final node reported that it can't be
+	 * reached, or in MPP payments there are no more paths we can
+	 * attempt. Modifiers need to leave failures alone once this is set to
+	 * true. Set only on the root payment. */
+	bool abort;
```

Comment:
is `abort` a keyword? it's definitely a stdlib method name. (github's syntax highlighter is showing it as a keyword)

Reply:
Indeed, it seems to be a function name defined in `stdlib.h`. Not sure what the downsides of re-using the name, but I'll look for something else.